src/api/job.js

```diff
@@ -141,4 +141,31 @@ router.get('/', async (request, response) => {
   }
 });
 
+router.get('/pagination', async (request, response) => {
```

Comment:
We shouldn't create a new endpoint for pagination. We should use the existed API endpoint when we return all jobs `GET /api/jobs`. In the real project, it's rare to return all data in our database, right?